src/umpire/strategy/FixedPool.inl

```diff
@@ -77,20 +78,23 @@ FixedPool<T, NP, IA>::FixedPool(
 
 template <typename T, int NP, typename IA>
 FixedPool<T, NP, IA>::~FixedPool() {
-    for (struct Pool *curr = m_pool; curr; ) {
-      struct Pool *next = curr->next;
-      m_allocator->deallocate(curr);
-      m_current_size -= sizeof(T)*m_num_per_pool;
-      curr = next;
-    }
+  for (struct Pool *curr = m_pool; curr; ) {
```

Comment:
@davidbeckingsale I'm going to leave this in because the destructor previously existed.